fullstack/server/config/db.config.ts

```diff
@@ -0,0 +1,17 @@
+import { Options } from "sequelize"; 
+
+const dbConfig: Options = {
+  username: 'root',
+  password: 'abc123xyz',
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- a6b2ca0d-009e-4d82-a0e7-1843801cbd59 -->